src/nft/components/bag/profile/ListingModal.css.ts

```diff
@@ -29,20 +29,18 @@ export const sectionDivider = style([
 
```

Comment:
Acknowledging that I'm still using VE here. Once the polish tickets are complete I plan on re-writing this component using styled components

Reply:
based acknowledgment